common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -126,3 +127,41 @@ def get_values(self, key_list):
             result_map[key] = val
 
         return result_map
+
+    def get_settings_names(self):
+        """
+        Returns all settings displayed on the advanced settings page/screen/modal/whatever
+        We call it 'name', but it's really whatever is embedded in the 'id' element for each field
+        """
+        query = self.q(css=SETTINGS_NAME_SELECTOR)
+        return [match.get_attribute('id') for match in query]
```

Comment:
What I mean is something along these lines (I didn't test it out for syntax etc):
```
return self.q(css=SETTINGS_NAME_SELECTOR).attrs('id')
```